Userland/Libraries/LibManual/SectionNode.cpp

```diff
@@ -40,20 +42,36 @@ ErrorOr<void> SectionNode::reify_if_needed() const
         return {};
     m_reified = true;
 
-    Core::DirIterator dir_iter { TRY(path()).to_deprecated_string(), Core::DirIterator::Flags::SkipDots };
+    auto own_path = TRY(path());
+    Core::DirIterator dir_iter { own_path.to_deprecated_string(), Core::DirIterator::Flags::SkipDots };
+
+    struct Child {
+        NonnullRefPtr<Node> node;
+        String name_for_sorting;
+    };
+    Vector<Child> children;
 
-    Vector<String> page_names;
     while (dir_iter.has_next()) {
         LexicalPath lexical_path(dir_iter.next_path());
-        if (lexical_path.extension() != "md")
-            continue;
-        page_names.append(TRY(String::from_utf8(lexical_path.title())));
+        if (lexical_path.extension() != "md") {
+            dbgln("Non-md inode {}", lexical_path);
```

Comment:
This dbgln and the one below will spam the console when browsing the man pages as-is, right? can we hide them behind a debug flag or remove?

Reply:
I will just remove them, it was to make sure that DirIterator picked up on exactly the things it should.